app/assets/images/payment_icons/abitab.svg

```diff
@@ -0,0 +1,6 @@
+<svg version="1.1"
```

Comment:
Please read the [contributions guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

- Remove any unused `id` attributes
- Remove any unused `class` attributes
- `viewBox` value should be `0 0 38 24`
- Run the svg code through `svgo`